ChangeLog.md

```diff
@@ -1,3 +1,10 @@
+## 0.4.0 - YYYY-MM-DD
```

Comment:
Is there a reason this is now marked as a breaking change?

Reply:
My thinking was this: the Haskell PVP decision tree says a major bump is necessary if the 'behaviour' of any exported function changes. Here, `untarChunks` used to produce all `ustar` chunks (including those chunks used to specify pax header blocks and extended headers) and now it produces the chunks that remain after the application of any pax header blocks and extended headers. That seemed to me to be a 'change in behaviour' rather than just a 'bug fix', and is why I export `untarChunksRaw` (in order to preserve the previous behaviour for people who may actually have needed it).

Reply:
I hear that. I'd interpret it differently. If there was a function that would previously fail on some input but now can successfully parse it, that's an improvement, not a breaking change. Maybe that isn't the case here. Either way, I'll defer to your judgement on whether a major version bump is warranted. Want me to merge and release?

Reply:
@snoyberg, if you could merge and release, that would be good.